webapp/src/components/HomePage/PollCards/PollCards.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react'
+import { HeaderMenu, Header, Button, Card, Icon } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import Token from 'components/Token'
+import { Props } from './PollCards.types'
+import './PollCards.css'
+import { isFinished } from 'modules/poll/utils'
+import { distanceInWordsToNow } from '@dapps/lib/utils'
+import { PollWithAssociations } from 'modules/poll/types'
+
+export default class PollCards extends React.PureComponent<Props> {
+  getHandler = (poll: PollWithAssociations, onClick: Function) => () =>
+    onClick(poll)
+  render() {
+    const { polls, title, meta, onClick, onViewMore } = this.props
+    return (
+      <div className="PollCards">
+        <HeaderMenu>
+          <HeaderMenu.Left>
+            <Header>{title}</Header>
+          </HeaderMenu.Left>
+          <HeaderMenu.Right>
+            <Button basic size="small" onClick={onViewMore}>
+              {t('global.view_more')}
+              <Icon name="chevron right" />
+            </Button>
+          </HeaderMenu.Right>
+        </HeaderMenu>
+        <Card.Group>
+          {polls.map(poll => (
+            <Card
+              link
+              onClick={this.getHandler(poll, onClick)}
```

Comment:
passing onClick as param shouldn't be necessary because it's available on `this.props`

Also maybe getHandler should be `getPollClickHandler` ?